Resources/Prototypes/Entities/Structures/Walls/walls.yml

```
@@ -672,6 +672,8 @@
     damageModifierSet: Metallic
   - type: Physics
     bodyType: Static
+  - type: Reflect
+    reflectProb: 0.1
```

```suggestion
    reflectProb: 1
```

Shuttle walls should reflect 100% of the time.